packages/@lwc/engine/src/framework/main.ts

```diff
@@ -25,6 +25,6 @@ export { default as track } from './decorators/track';
 export { default as readonly } from './decorators/readonly';
 export { default as wire } from './decorators/wire';
 export { default as decorate } from './decorators/decorate';
-export { buildCustomElementConstructor } from './wc';
+export { deprecatedBuildCustomElementConstructor as buildCustomElementConstructor } from './wc';
```

Comment:
Renaming the export is a breaking change by itself and defeats the purpose of the deprecation. I would recommend reverting this change.

Reply:
no, this is re-exporting something from another module by renaming it to its old name, that's all, this is not a breaking change.

Reply:
Oh I see, I read it the other way around.